pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "icepyx"
+description = "Python tools for obtaining and working with ICESat-2 data"
+license = {file = "LICENSE"}
+readme = "README.rst"
+
+requires-python = "~=3.7"
```

Comment:
Oh shoot, totally overlooked this line. This should have been `>=3.7` instead...

Reply:
On it!